resources/assets/components/utilities/SitewideBanner/SitewideBanner.js

```diff
@@ -1,9 +1,28 @@
+import { get } from 'lodash';
+import gql from 'graphql-tag';
 import { createPortal } from 'react-dom';
+import { useQuery } from '@apollo/react-hooks';
 import React, { useRef, useEffect } from 'react';
 
 import excludedPaths from './config';
+import { getUserId } from '../../../helpers/auth';
 import SitewideBannerContent from './SitewideBannerContent';
 
+const VOTER_REGISTRATION_STATUS = gql`
+  query VoterRegSiteWideBannerQuery($userId: String!) {
```

Comment:
I think we can downcase the w here:
```suggestion
  query VoterRegSitewideBannerQuery($userId: String!) {
```